IdentityCore/src/broker_operation/request/MSIDBaseBrokerOperationRequest.h

```diff
@@ -28,6 +28,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface MSIDBaseBrokerOperationRequest : NSObject
 
 @property (nonatomic, class, readonly) NSString *operation;
+@property (nonatomic) NSUUID *correlationId;
```

Comment:
Could you kindly enlighten me on why it cannot be nullable any longer?

Reply:
Did I remove the attribute, I thought I add a new property, not removing anything in this class :D

Reply:
But I think it should not be nullable, cause we already assigned the value in 2 child subclass init. Do you think so?

Reply:
Should this be readonly?

Reply:
I think it shouldn't because we move the correlationId to the base class `MSIDBaseBrokerOperationRequest`, however one of the child (**MSIDBrokerOperationRequest**) we actually fill the value from external `[MSIDBrokerOperationRequest fillRequest:keychainAccessGroup:clientMetadata:context]` so the natual of this property is not read-only. 
Will that make sense? @oldalton @NerevarineRule

Reply:
ok, I see. Thanks for clarifying.